rover/src/main/kotlin/io/rover/rover/platform/LocalStorage.kt

```diff
@@ -0,0 +1,39 @@
+package io.rover.rover.platform
+
+import android.content.Context
+import android.content.Context.MODE_PRIVATE
+
+/**
+ * Very simple hash-like storage of keys and values.
+ */
+interface KeyValueStorage {
```

Comment:
Do you need both the LocalStorage and KeyValueStorage interfaces? Could they be merged?

Reply:
On Android you can create as many shared preferences instances as you like (and they're each given an identifier).  This interface is meant to be a facade around that.  This means different Rover services can have their own separate storage area, and avoid key collisions between them.

Merging them together is fine if we would prefer to have a single shared preferences amongst the Rover services (which would be fine too).

Reply:
I don't think we need to have separate shared preferences between services, can't think of a use-case for that. Your call if you want to merge these or keep them separate.

Reply:
hmm, Having them all together does make clear/reset/logout a little simpler.